File: Golang/maxConsectiveOnes.go
func findMaxConsecutiveOnes(nums []int) int {
    var lastMaxConsecNum = 0 // Remembers last Maximum Consecutive number
    var currMaxConsecNum = 0 // Remembers current Maximum Consecutive number
    
    for idx := 0; idx < len(nums); idx++ {
        if nums[idx] == 1{
            currMaxConsecNum+=1
            
            // Check if Current Consecutive number is greater Last Consecutive number
            if currMaxConsecNum > lastMaxConsecNum{
                lastMaxConsecNum = currMaxConsecNum
            }
        } else{
            // Reset Current Consecutive number
            currMaxConsecNum = 0
        }
	}
    return lastMaxConsecNum
}

File: Python/max_consective_ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        LastMaxConsecNum = 0  # Remembers last Maximum Consecutive number
        CurrMaxConsecNum = 0  # Remembers current Maximum Consecutive number
        
        for idx in range(0, len(nums)):
            if nums[idx] == 1:
                CurrMaxConsecNum+=1
                # Check if Current Consecutive number is greater Last Consecutive number
                if CurrMaxConsecNum > LastMaxConsecNum:
                    LastMaxConsecNum = CurrMaxConsecNum
            else:
                # Reset Current Consecutive number
                CurrMaxConsecNum = 0
        
        return LastMaxConsecNum